index.bs

```diff
@@ -2916,6 +2916,252 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method ### {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(optional PublicKeyCredentialSignalOptions options = {});
+    };
+
+    dictionary PublicKeyCredentialSignalOptions {
+        UnknownCredentialIdReport              unknownCredential;
+        CurrentCredentialsReport               currentCredentials;
+    };
+
+    dictionary UnknownCredentialIdReport {
+        USVString                              rpId;
```

Comment:
I think it'd be easier for an RP dev to understand required vs optional values if we went ahead and explicitly made this required:
```suggestion
        required USVString                              rpId;
```

Reply:
Done.